life_expectancy/cleaning.py

```diff
@@ -0,0 +1,67 @@
+import os
+import numpy as np
+import pandas as pd
+import argparse
+
+
+# Determines the absolute path of the directory we are working on 
+file_dir = os.path.dirname(os.path.abspath(__file__))
+
+# Determines the relative path of the directory we are working on
+data_dir = os.path.join(file_dir, 'data')
+
+
+def split_column(df_: pd.DataFrame) -> pd.DataFrame:
+    """splits a column into several, based on a comma separating diff values"""
+    df = df_.copy()
+    new_cols = df['unit,sex,age,geo\\time'].str.split(',', expand=True)
+    new_cols.columns = ['unit', 'sex', 'age', 'region']
+    df = pd.concat([df, new_cols], axis=1)
+    df = df.drop(['unit,sex,age,geo\\time'], axis=1)
+    return df
+
+
+def extract_numeric_values_from_column(df_: pd.DataFrame, column: str) -> pd.DataFrame:
+    """extracts numerical values from a string including several strange characteres"""
+    df = df_.copy()
+    df[column]= df[column].replace(to_replace=r'[^\d\.]+', value='', regex=True)
+    df[column] = df[column].replace(to_replace='', value=np.nan)
+    df[column] = pd.to_numeric(df[column], errors='coerce')
+    return df
+
+def load_data() -> pd.DataFrame:
+    
+    # Load the raw data from a TSV file
```

Comment:
Maybe the comment can be in a function docstring instead

Reply:
Thank you Phuc!!! Done